docs/asciidoc/static/deploying.asciidoc

```diff
@@ -64,7 +64,7 @@ nodes. By default, Logstash uses the HTTP protocol to move data into the cluster
 You can use the Elasticsearch HTTP REST APIs to index data into the Elasticsearch cluster. These APIs represent the 
 indexed data in JSON. Using the REST APIs does not require the Java client classes or any additional JAR 
 files and has no performance disadvantages compared to the transport or node protocols. You can secure communications 
-that use the HTTP REST APIs with the {shield}[Shield] plugin, which supports SSL and HTTP basic authentication.
+that use the HTTP REST APIs with the {shield-13}[Shield] plugin, which supports SSL and HTTP basic authentication.
 
```

Comment:
Are you sure about this formatting? I use attributes to sub in the name value I want, and have no need for the assigned name `[Shield]`.  I wanted to test this, but I couldn't check out 1.5 for some reason.

Reply:
New thought...  Maybe this old branch doesn't know how/where to resolve {docs-root}?

Reply:
Yeah, it's a link. In a couple places we do `{shield-13}/something.html[Link Text]` but here we just go to the root of the Shield documentation.

Reply:
> Maybe this old branch doesn't know how/where to resolve {docs-root}?

[This code](https://github.com/elastic/docs/blob/69e747baa8653201b3589869bb2ab9c9dc2e7b87/lib/ES/Util.pm#L349-L359) is where the various `-root` attribute get injected automatically, via the command line. It wasn't getting added before I added `docs` to `sources` in elastic/docs#1922, but now it _should_ be working.